.flake8

```diff
@@ -1,7 +0,0 @@
-[flake8]
-exclude = ./docs/*, */migrations/*, ./pulpcore/plugin/docs/*
-ignore = W503,Q000,Q003,D100,D104,D106,D200,D202,D205,D400,D401,D402
-# E203: whitespace before ':'; https://github.com/psf/black/issues/279
-# E401: multiple imports on one line
-extend-ignore = E203,E401
-max-line-length = 100
```

Comment:
Looks like flake is now unhappy, possibly because of this change?:
```
flake8
.ci/scripts/check_click_for_mypy.py
MYPYPATH=pulp-glue-deb mypy
tests/conftest.py:9: error: Function is missing a type annotation  [no-untyped-def]
tests/test_help_pages.py:7: error: Skipping analyzing "pulp_cli": module is installed, but missing library stubs or py.typed marker  [import-untyped]
tests/test_help_pages.py:7: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
Found 2 errors in 2 files (checked 8 source files)
```

Reply:
That's mypy being unhappy. conftest needs some annotation love (I suggest adopting from pulp-cli-gem).
And the other error is the pulp_cli module missing a py.typed file. I thought that was fixed and released on the pulp-cli side.